src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -124,18 +128,72 @@ private InlineDiffLine<CodeLine>[] CreateDiffOnlyLines(InlineDiffLine<CodeLine>[
         {
             var filteredLines = new List<InlineDiffLine<CodeLine>>();
             int lastAddedLine = -1;
+
+            // Keep track of any enclosing context and the level of indent the
+            // deepest corresponds to.  This allows us to include the enclosing
+            // classes, namespaces, etc. when viewing only the diff.  This is
+            // best effort that presumes significant indentation.
+            var enclosingContext = new List<(int, string)>();
+            int enclosingContextIndent = 0;
+            var enclosingMarkup = new StringBuilder(capacity: 256);
+
             for (int i = 0; i < lines.Count(); i++)
             {
+                // Compare the current line's indent to the enclosing context
+                (int indentSize, _) = GetIndentAndCode(lines[i].Line.DisplayString);
+                if (indentSize > enclosingContextIndent && i > 0)
+                {
+                    // If this line is indented further, then the previous line
+                    // should be part of the enclosing context
+                    (_, string code) = GetIndentAndCode(lines[i - 1].Line.DisplayString);
+                    enclosingContext.Add((i - 1, code.Trim()));
```

Comment:
It's throwing null reference error in this line on staging instance. I think code is null when line is empty.

~~~System.NullReferenceException:
   at APIViewWeb.Pages.Assemblies.ReviewPageModel.CreateDiffOnlyLines (APIViewWeb, Version=1.0.0.0, Culture=neutral, PublicKeyToken=7a84984c17ca6b7dAPIViewWeb, Version=1.0.0.0, Culture=neutral, PublicKeyToken=7a84984c17ca6b7d: D:\a\1\s\src\dotnet\APIView\APIViewWeb\Pages\Assemblies\Review.cshtml.csAPIViewWeb, Version=1.0.0.0, Culture=neutral, PublicKeyToken=7a84984c17ca6b7d: 149)
   at APIViewWeb.Pages.Assemblies.ReviewPageModel.CreateLines

Reply:
What language, out of curiosity?

Reply:
https://apiviewstaging.azurewebsites.net/Assemblies/Review/73f4a17f8dff41dca9762ad38870d475?diffRevisionId=603455b588e44bed9f0ae0dbeaedd393&doc=False&diffOnly=True